samples/ibm-samples/ffdl-seldon/credentials/creds.ini

```diff
@@ -0,0 +1,8 @@
+[CREDENTIALS]
```

Comment:
IP addresses and secrets are sensitive information and should ideally not be checked in.

Reply:
We will remove creds.ini